Loop/Views/SettingsView.swift

```diff
@@ -89,19 +90,22 @@ extension SettingsView {
         
     private var therapySettingsSection: some View {
         Section(header: SectionHeader(label: NSLocalizedString("Configuration", comment: "The title of the Configuration section in settings"))) {
-            return NavigationLink(destination: TherapySettingsView(
-                viewModel: TherapySettingsViewModel(mode: .settings,
-                                                    therapySettings: viewModel.therapySettings,
-                                                    supportedInsulinModelSettings: viewModel.supportedInsulinModelSettings,
-                                                    pumpSupportedIncrements: viewModel.pumpSupportedIncrements,
-                                                    syncPumpSchedule: viewModel.syncPumpSchedule,
-                                                    chartColors: .primary,
-                                                    didSave: viewModel.didSave))) {
-                LargeButton(action: { },
+            LargeButton(action: { self.showTherapySettings = true },
```

Comment:
Want to check it's intentional that this screen is now presented modally

Reply:
Yes.  This is very intentional ... based on Design feedback: they do not want some screens to show as a "left-right slide" navigation (TherapySettings) and others as a sheet (modal).  This makes them consistent.